### ISSUE TITLE
Microsoft authentication url is open in internal webkit browser instead of default browser and fails

### ISSUE THREAD

**Issue by FranzHeubach** [FTBTeam/FTB-App#763]
**Operating System**: Fedora 36

**App Version**: 202211151621-329441ed6d-release
**UI Version**: 329441ed6d

# Description
1. Open the FTB app
2. Press log in button on the bottom of the left side pane
3. Press log in with microsoft account in the pop up window
4. This opens another popup internal to the app called FTB Minecraft Authenticator which fails to communicate back to the application.

# Expected behaviour
I expect that instead of the pop up, that the url is opened in my default system browser so that I could complete the authentication process. This is the process that others users experience.

# Work around
TLDR: Leave the pop-up open and navigate to `msauth.feed-the-beast.com` manually. Complete the authentication process and then close the tab and the pop-up.

I played around with the pop up window until I found that alt would give me a menu bar. Within the menu bar I found the developer tools. I then found the URL that the pop-up was trying to open: msauth.feed-the-beast.com and opened it manually in my browser. This seemed to provide a work around.

**Comment by MichaelHillcox** [comment]
This should be resolved :+1:
